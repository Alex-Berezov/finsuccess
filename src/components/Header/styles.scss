.header {
  display: flex;
  justify-content: space-between;
  padding: 10px 0;

  .nav {
    display: flex;

    p {
      margin-left: 15px;
    }
  }
}