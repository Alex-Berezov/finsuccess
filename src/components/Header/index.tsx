import React from 'react';
import './styles.scss'

const Header = () => {
  return (
    <div className='header'>
      <p>Logo</p>
      <div className="nav">
        <p>Item 1</p>
        <p>Item 1</p>
        <p>Item 1</p>
      </div>
    </div>
  );
};

export default Header;