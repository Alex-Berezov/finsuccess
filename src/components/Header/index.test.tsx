import Header from './index';
import { render } from '@testing-library/react';

describe('Header tests', () => {
  const { getByTestId, rerender } = render(<Header />)
  it('Test Header is mounted', () => {
    expect(getByTestId('header')).toBeInTheDocument()
    expect(getByTestId('logo')).toBeInTheDocument()
    expect(getByTestId('nav')).toBeInTheDocument()
    expect(getByTestId('header')).toMatchSnapshot()
  })
})