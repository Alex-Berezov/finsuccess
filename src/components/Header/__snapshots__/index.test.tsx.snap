// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header tests Test Header is mounted 1`] = `
<div
  class="header"
  data-testid="header"
>
  <p
    data-testid="logo"
  >
    Logo
  </p>
  <div
    class="nav"
    data-testid="nav"
  >
    <p>
      Item 1
    </p>
    <p>
      Item 1
    </p>
    <p>
      Item 1
    </p>
  </div>
</div>
`;
